### PR TITLE
LPS-131313 | master

### PR DESCRIPTION
**Pull request by WYZ-Issac**
https://issues.liferay.com/browse/LPS-131313

This is only for master.

@Tim-Cao

### CODE REVIEW

**Comment by WYZ-Issac** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 1]
```diff
@@ -209,6 +209,22 @@ definition {
 		}
 	}
 
+	@description = "Ensure that the URL is correct when there is an Asset Publisher in a page."
+	@priority = "3"
+	test CheckTheUrl {
+		task ("Check the URL is correct.") {
+			Navigator.openSiteURL(
+				baseURL = "http://localhost:8080",
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:217]: Shouldn't hardcode

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase [hunk 2]
```diff
@@ -209,6 +209,22 @@ definition {
 		}
 	}
 
+	@description = "Ensure that the URL is correct when there is an Asset Publisher in a page."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/asset/assetpublisher/AssetPublisherUseCase.testcase:212]: @description = "This is a test for LPS-128196. View the URL when there is an Asset Publisher in a page."

**Comment by Tim-Cao** [comment]
ci:close

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/6007

**Pull request closed without merging**